<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingForm</class>
 <widget class="QWidget" name="SettingForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Log Directory:</string>
       </property>
      </widget>
      <widget class="QToolButton" name="toolBtnLogDir">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>50</y>
         <width>25</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="editLogDir">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>321</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>log</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Process Filter</string>
      </attribute>
      <widget class="QListWidget" name="listProcess">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>341</width>
         <height>192</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>eclipse</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>javaw</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>chrome</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>firefox</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>iexplore</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>devenv</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>WINWORD</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>EXCEL</string>
        </property>
       </item>
      </widget>
      <widget class="QCheckBox" name="checkBoxProc">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>131</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Record All Processes</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnAddProc">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>230</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Add </string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnDelProc">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>230</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Collected Data</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Mouse</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="cbAllMouse">
            <property name="text">
             <string>All Mouse Event</string>
            </property>
            <property name="autoExclusive">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbMouseClick">
            <property name="text">
             <string>Only Mouse Click</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Keyboard</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QCheckBox" name="cbKeyinput">
            <property name="text">
             <string>Keyboard Input</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Clipboard</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QCheckBox" name="cbCopy">
            <property name="text">
             <string>Copy Event(only text)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Screen-capture</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QCheckBox" name="cbSCNever">
            <property name="text">
             <string>Never</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="cbSCWindow">
            <property name="text">
             <string>For New Window</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="cbSCEvery">
            <property name="text">
             <string>Every Important Event</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
